tests/container_disk_test.go

```diff
@@ -175,13 +175,33 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 
 	Describe("[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]Starting from custom image location", func() {
 		Context("with disk at /custom-disk/downloaded", func() {
+
 			It("[test_id:1466]should boot normally", func() {
-				vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirrosCustomLocation), "#!/bin/bash\necho 'hello'\n")
-				for ind, volume := range vmi.Spec.Volumes {
-					if volume.ContainerDisk != nil {
-						vmi.Spec.Volumes[ind].ContainerDisk.Path = "/custom-disk/downloaded"
+				const (
+					minimalVMIRequiredMemory = "256Mi"
+					volName                  = "disk0"
+					path                     = "/custom-disk/downloaded"
+				)
+
+				var withVolumePath libvmi.Option = func(vmi *v1.VirtualMachineInstance) {
```

Comment:
I would prefer to see this as a static function. The indentation here is too deep.